src/jit/lsra.cpp

```diff
@@ -4070,6 +4075,15 @@ void LinearScan::buildRefPositionsForNode(GenTree*                  tree,
             useCandidates  = allRegs(registerType);
         }
 
+#ifdef _TARGET_ARM_
+        if (tree->OperIsPutArgStruct())
+        {
+            // get i-th candidate
+            currCandidates = genFindLowestReg(candidates);
```

Comment:
We cannot compare registers return by `GetRegNumByIdx` here because it is not initialized here (LSRA buildInterval phase). It is initialized at LSRA resolve phase.